docs/css.md

```diff
@@ -298,11 +291,13 @@ yarn add --dev typings-for-css-modules-loader
 const { environment } = require('@rails/webpacker')
 
 // replace css-loader with typings-for-css-modules-loader
-environment.loaders.get('moduleSass').use = environment.loaders.get('moduleSass').use.map((u) => {
-  if(u.loader == 'css-loader') {
-    return { ...u, loader: 'typings-for-css-modules-loader' };
-  } else {
-    return u;
-  }
-});
+environment.loaders.get('moduleSass').use = environment.loaders
```

Comment:
Hey @gauravtiwari 

should it be `loaders.get('moduleSass').use` ?

```
const { environment, loaders } = require('@rails/webpacker')

console.log('environment', environment)
console.log('loaders', loaders)

environment undefined
loaders [
  {
    test: [
```

Reply:
@pustovalov Hey, I think we need to overhaul the documentation. The class-based API is removed in 6.x

I will work on updating docs.